_data/navigation.yml

```diff
@@ -98,25 +96,27 @@ authors_perpetual_reviews.md:
     - title: Revision schedule for perpetual reviews
       url: /authors/perpetual_reviews/index.html#revision-schedule-for-perpetual-reviews
 
-authors_compare_simulations.md:
-  - title: Comparisons of Molecular Simulation Packages
-    url: /authors/compare_simulations/index.html#
+authors_best_practices.md:
```

Comment:
Not sure if this is a mix-up: gihub shows compare_sim being changed to best_practices.
Likewise below

Reply:
Looks like an issue with how the navigation processing scripts orders things - it put the software analysis before the other items, and it's unclear why.  Perhaps @ptmerz can take a quick look when he has a chance.